python/mage/node_classification/utils/metrics.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import mgp
```

Comment:
This is red flag for me. I think we shouldn't promote mgp to utils modules. Purpose of such behaviour is to unit test it and be independent of memgraph.

Reply:
Try changing it, to remove mgp from this file and push logic to python/node_classification.py

Reply:
Thanks, value comment! Repaired.